### PR TITLE
[DO NOT MERGE] Complete Rework (Experimental)

### PR DESCRIPTION
**Pull request by ZeitHeld**
This is an **_experimental_** Rework of the current, awkwardly made website

it will totally change from time to time and probably develop as well

You can visit the rework page [here](https://zeitheld.github.io/OoTxMM-Item-Sanity-Checklist/)

**CURRENT FEATURES:**
- Checkboxes that can be ticked
- A Header with the Title that sticks at the top
- 2 Dropdown Menus for the Sanity-Type and Sanity Region
- Working Tooltips that Explain the Drop downs from above

I also left some space and ideas for other sanities as well

There is <ins>currently</ins> **_NO JAVASCRIPT_** included, but **will be implemented later**, to create better formatting and make any buttons do anything as well.

**Providing any feedback or ideas to this would be lovely**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                <div class="area-sanities">
+                    <div class="sanity">
+                        <div class="drop-name">
+                            <label for="sanity">Sanity Type:</label>
+                            <div class="hint">
+                                <label>?</label>
+                                <div class="tooltip">
+                                    <p><strong>Scrubs: </strong>Each Deku Scrub that do a one-time trade</p>
+                                    <p><strong>Shops: </strong>Each Shop has a total of 8 completely random items</p>
+                                    <p><strong>Cows: </strong>Each Shop will provide a random Item when <strong>Epona&apos;s Song</strong> has been played</p>
```
Comment (ScienceNerd2240) [inline index.html]: small typo, should read "Each Cow" instead of shop

Comment (ZeitHeld) [reply index.html]: oh bruh, I did not notice
will change it with the next update

Comment (ScienceNerd2240) [reply index.html]: i'm merging this in now, i have the main branch backed up to a new branch and set that one to the default branch for the repo. once that is done, ill rename the branches again

**Comment by ScienceNerd2240** [comment]
I went ahead and made the change real quick, before merging. you should still be able to work on it after I merge and whatnot

**Pull request merged**